examples/running-tests-using-travis-and-saucelabs/.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: node_js
```

Comment:
Example project missing README which will describe how to run this project, one-liner description on what this project is about with cross-reference to recipe